src/citrine/informatics/executions/sample_design_space_execution.py

```diff
@@ -0,0 +1,101 @@
+from functools import partial
+from typing import Optional, Iterable
+from uuid import UUID
+
+from citrine._rest.asynchronous_object import AsynchronousObject
+from citrine._rest.pageable import Pageable
+from citrine._rest.paginator import Paginator
+from citrine._rest.resource import Resource
+from citrine._serialization import properties
+from citrine._session import Session
+from citrine._utils.functions import format_escaped_url
+from citrine.informatics.design_candidate import SampleSearchSpaceResultCandidate
+
+
+class SampleDesignSpaceExecution(
+    Resource["SampleDesignSpaceExecution"], Pageable, AsynchronousObject
+):
+    """The execution of a Sample Design Space task.
+
+    Possible statuses are INPROGRESS, SUCCEEDED, and FAILED.
+    Design executions also have a ``status_description`` field with more information.
```

Comment:
Update name.

Reply:
I missed this comment on my last push. But I have a local update to the doc string as
```
    """The execution of a Sample Design Space task.

    Possible statuses are EXECUTING, SUCCEEDED, and FAILED.
    Sample Design Space executions also have a ``status_detail`` field with more information.

    """
```

Was this what you were pointing out?